src/analyzing/Constant_propagation.ml

```diff
@@ -623,6 +623,20 @@ let add_special_constants env lang prog =
 (* Entry point *)
```

Comment:
We need a changelog entry to announce that Pro will recognize constants defined via macros.

Reply:
(You should always follow the PR checklist.)

Reply:
Done.